Resources/Prototypes/Entities/Clothing/Head/hats.yml

```
@@ -85,7 +85,7 @@
       sprite: Clothing/Head/Hats/casa.rsi
 
 - type: entity
-  parent: ClothingHeadBase
+  parent: [ClothingHeadBase, BaseRestrictedContraband]
```

```suggestion
  parent: [ClothingHeadBase, BaseCommandContraband]
```
I think you meant to put them into Command contraband.

---

I meant to put it into science contraband, thanks for the catch.